### PR TITLE
Use clap for CLI

### PR DESCRIPTION
**Pull request by MichaelGrupp**
Simple change to get a nicely formatted --help for free and to allow more arguments in the future.

### CODE REVIEW

**Comment by MichaelGrupp** [comment]
```
$ termviz --help
termviz 
ROS visualization on the terminal

USAGE:
    termviz [config]

ARGS:
    <config>
            Optional YAML file with a custom termviz configuration.

OPTIONS:
    -h, --help
            Print help information

More documentation can be found at: github.com/carzum/termviz
```

The last line is a lie, but this will change with the README.

File: Cargo.toml [hunk 1]
```diff
@@ -11,6 +11,7 @@ exclude = ["assets/*", ".github"]
 approx = "0.5.1"
 byteorder = "*"
 chrono = "0.2.25"
+clap = "*"
```
Comment (MichaelGrupp) [inline Cargo.toml:14]: I don't know if this needs a follow up in the catkin build?

Comment (carzum) [reply Cargo.toml:14]: nope

File: src/main.rs [hunk 2]
```diff
@@ -15,18 +15,25 @@ use std::collections::HashMap;
...
     // Terminal initialization
 
-    let args: Vec<String> = env::args().collect();
-    let conf = config::get_config(args).unwrap();
+    let matches = Command::new("termviz")
+        .about("ROS visualization on the terminal")
+        .arg(
+            Arg::new("config").long_help("Optional YAML file with a custom termviz configuration."),
+        )
+        .after_help("More documentation can be found at: github.com/carzum/termviz")
```
Comment (carzum) [inline src/main.rs:33]: putting https:// infront will increase the chance of terminals interpreting it as a clickable link

**Pull request merged**